de.po

```diff
@@ -2198,15 +2198,15 @@ msgstr ""
 
 #: src/components/create-vm-dialog/createVmDialog.jsx:687
 msgid "This volume is already used by another VM."
-msgstr ""
+msgstr "Dieser Datenträger ist bereits in Verwendung einer anderen VM."
```

Comment:
This sounds a bit weird IMHO. Maybe "Dieser Datenträger wird bereits von einer anderen VM verwendet."?